src/renderer/hub.css

```
@@ -118,12 +144,201 @@ body {
+
+.key-input:focus {
+  border-color: var(--glow-dim);
+  box-shadow: 0 0 0 1px var(--glow-faint);
+}
+
+.save-btn {
+  background: var(--surface-raised);
+  border: 1px solid var(--border);
+  border-radius: 4px;
+  color: var(--text-dim);
+  font-family: 'Rajdhani', sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Rajdhani" (font-family-name-quotes)

(font-family-name-quotes)